docs/en_US/Compressor/Pruner.md

```diff
@@ -390,5 +395,170 @@ You can view example for more information
+pruner.compress()
+```
+
+You can view [example](https://github.com/microsoft/nni/blob/master/examples/model_compress/auto_pruners_torch.py) for more information.
+
+#### User configuration for SimulatedAnnealing Pruner
+
+- **sparsity:** How much percentage of convolutional filters are to be pruned.
+- **op_types:** "Conv2d" or "default".
+- **evaluator:** Function to evaluate the masked model. This function should include `model` as the only parameter, and returns a scalar value.
+- **optimize_mode:** Optimize mode, 'maximize' or 'minimize', by default 'maximize'.
+- **base_algo:** base pruning algorithm. 'level', 'l1' or 'l2', by default 'l1'.
+- **start_temperature:** Simualated Annealing related parameter.
+- **stop_temperature:** Simualated Annealing related parameter.
+- **cool_down_rate:** Simualated Annealing related parameter.
+- **perturbation_magnitude:** Initial perturbation magnitude to the sparsities. The magnitude decreases with current temperature.
+- **experiment_data_dir:** PATH to save experiment data.
+
+
+## ADMM Pruner
+Alternating Direction Method of Multipliers (ADMM) is a mathematical optimization technique,
+by decomposing an original problem into two subproblems that can be solved separately.
+In weight pruning problem, the two subproblems are solved via gradient descent algorithm and Euclidean projection respectively.
+This solution framework applies both to non-structured and different variations of structured pruning schemes.
+
+For more details, please refer to [A Systematic DNN Weight Pruning Framework using Alternating Direction Method of Multipliers](https://arxiv.org/abs/1804.03294).
+
+#### Usage
+
+PyTorch code
+
+```python
+from nni.compression.torch import ADMMPruner
+config_list = [{
+            'sparsity': 0.8,
+            'op_types': ['Conv2d'],
+            'op_names': ['conv1']
+        }, {
+            'sparsity': 0.92,
+            'op_types': ['Conv2d'],
+            'op_names': ['conv2']
+        }]
+pruner = ADMMPruner(model, config_list, trainer=trainer, optimize_iterations=30, epochs=5)
+pruner.compress()
+```
+
+You can view [example](https://github.com/microsoft/nni/blob/master/examples/model_compress/auto_pruners_torch.py) for more information.
+
+#### User configuration for ADMM Pruner
+
+- **sparsity:** How much percentage of convolutional filters are to be pruned.
+- **op_types:** Currently only Conv2d is supported in ADMMPruner.
+- **trainer:** Function used for the first optimization subproblem.
+This function should include `model, optimizer, criterion, epoch, callback` as parameters, where callback should be inserted after loss.backward of the normal training process.
+- **optimize_iteration:** ADMM optimize iterations.
+- **training_epochs:** training epochs of the first optimization subproblem.
+- **row:** penalty parameters for ADMM training.
+- **base_algo:** base pruning algorithm. 'level', 'l1' or 'l2', by default 'l1'.
```

Comment:
why this one does not have `experiment_data_dir`?

Reply:
ADMMPruner is not an auto pruner, there is thus no experiment data generated. More explanation on what is included as experiment data added for the auto pruners.